api/config/packages/framework.yaml

```diff
@@ -16,17 +16,6 @@ framework:
     php_errors:
         log: true
 
-    session:
-        # enables the support of sessions in the app
-        enabled: true # sessions are needed for oauth state parameters
-        # ID of the service used for session storage.
-        # NULL means that Symfony uses PHP default session mechanism
-        handler_id: 'session.handler.native_file'
-        save_path: '%kernel.project_dir%/var/sessions/%kernel.environment%'
-        # improves the security of the cookies used for sessions
-        cookie_secure: 'auto'
-        cookie_samesite: 'lax'
-
```

Comment:
Only minor, but I think below `session:` key under `when@test:` we could also remove again

Reply:
Nice catch, thanks. Fixed.